WalletWasabi/Models/ChaumianCoinJoin/CcjRound.cs

```diff
@@ -720,8 +720,21 @@ private async Task TryOptimizeFeesAsync(IEnumerable<Coin> spentCoins)
 				int estimatedSigSizeBytes = UnsignedCoinJoin.Inputs.Count * Constants.P2wpkhInputSizeInBytes;
 				int estimatedFinalTxSize = UnsignedCoinJoin.GetSerializedSize() + estimatedSigSizeBytes;
 				Money fee = UnsignedCoinJoin.GetFee(spentCoins.ToArray());
+
 				// There is a currentFeeRate null check later.
-				FeeRate currentFeeRate = fee is null ? null : new FeeRate(fee, estimatedFinalTxSize);
+				FeeRate currentFeeRate = null;
+				if (fee is null)
+				{
+					Logger.LogError($"Cannot calculate CoinJoin transaction fee. (some spent coins are missing or {nameof(spentCoins)} is null)");
```

Comment:
`spentCoins` cannot be null. It'd throw an exception. (I take over this PR and merge it, because it's a nobrainer and about to conflict with something I'm working on.)